docs/upgrade.txt

```diff
@@ -61,6 +61,17 @@ version releases that introduced them. When upgrading library versions,
 address all the breaking changes between your current version and the
 planned upgrade version.
 
+.. _laravel-breaking-changes-v5.x:
+
+Version 5.x Breaking Changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This library version introduces the following breaking changes:
+
+- Removes support for the ``Auth\DatabaseTokenRepository`` class. Instead,
+  use the ``Illuminate\Queue\Failed\DatabaseFailedJobProvider`` class and
+  specify a connection to MongoDB.
```

Comment:
There's [another ticket](https://jira.mongodb.org/browse/DOCSP-43166?jql=project%20%3D%20DOCSP%20AND%20fixVersion%20%3D%20laravel-5.0) for the MongoFailedJobProvider class, but happy to include that in this PR since it's quick